src/FixedProductMarketMakerMapping.ts

```diff
@@ -117,10 +138,15 @@ function recordFPMMLiquidity(fpmm: FixedProductMarketMaker,
     } else {
       fpmmLiquidity.collateralTokenAmount = min(outcomeTokenAmounts);
     }
-    fpmmLiquidity.additionalLiquidityParameter = calculateLiquidityParameter(outcomeTokenAmounts);
+    fpmmLiquidity.additionalLiquidityParameter = calculateLiquidityParameter(
+      outcomeTokenAmounts
+    );
 
     fpmmLiquidity.sharesAmount = sharesAmount;
     fpmmLiquidity.collateralRemovedFromFeePool = collateralRemovedFromFeePool;
+    fpmmLiquidity.additionalSharesCost = max(outcomeTokenAmounts).minus(
+      min(outcomeTokenAmounts)
+    );
```

Comment:
Since my auto formatter basically updated this whole file, I should clarify that the only non-formatting changes made are on lines 147-162.